cli/src/graphlog.rs

```diff
@@ -132,12 +130,12 @@ pub fn node_template_for_key(
     key: &str,
     fallback: &str,
     ascii_fallback: &str,
-) -> String {
-    let symbol = settings.config().get_string(key);
-    if GraphStyle::from_settings(settings).is_ascii() {
-        symbol.unwrap_or_else(|_| ascii_fallback.to_owned())
+) -> Result<String, config::ConfigError> {
```

Comment:
Do you think it's worth mentioning this in the changelog as a breaking change?

Reply:
Added changelog entry.